.github/CODEOWNERS

```diff
@@ -193,6 +193,7 @@
 /pkg/version/                           @DataDog/agent-shared-components
 /pkg/obfuscate/                         @DataDog/agent-apm
 /pkg/trace/                             @DataDog/agent-apm
+/pkg/trace/api/otlp*.go                 @DataDog/opentelemetry
```

Comment:
(Not a blocker) Should this be also owned by APM or just us?

Reply:
@katiehockman what would you prefer here?

Reply:
(Didn't we redistribute responsibilities to _not_ own the code ? 👀 )

Reply:
Sure, we should own the OTLP specific bits, but, since this is part of the APM agent code after all, maybe you still want to co-own it, to ensure our implementation is aligned with the rest of the Trace Agent